File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/PullRequestListOptions.java
package com.tencent.devops.scm.api.pojo;

import com.tencent.devops.scm.api.enums.PullRequestState;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PullRequestListOptions {

    private PullRequestState state;
    private String sourceBranch;
    private String targetBranch;
    private Integer page;
    private Integer pageSize;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/PullRequestListOptions.kt
package com.tencent.devops.scm.api.pojo

import com.tencent.devops.scm.api.enums.PullRequestState

data class PullRequestListOptions(
    // PR/MR 状态
    val state: PullRequestState? = null,
    // 源分支
    val sourceBranch: String? = null,
    // 目标分支
    val targetBranch: String? = null,
    // 页码
    val page: Int? = null,
    // 每页数量
    val pageSize: Int? = null
)
